.drone.yml

```diff
@@ -381,6 +381,23 @@ steps:
     event:
       - push
 
+- name: upload_kernel_initrd_releases
+  image: plugins/github-release
+  settings:
+    prerelease: true
+    checksum:
+      - sha256
+    files:
+    - "dist/artifacts/harvester*initrd-amd64"
+    - "dist/artifacts/harvester*kernel-amd64"
```

Comment:
Seems we do not build the `harvester*kernel-amd64`?
Did you mean the `harvester*vmlinuz-amd64`?

For PXE boot, we may need vmlinuz and initrd.

Reply:
good spot.. let me fix it up